lib/sonos_nfc.js

```diff
@@ -60,6 +60,13 @@ const sonos_nfc = (nfc) => {
               case 'text':
                 const received_text = record.text;
                 console.log('Read from NFC tag with message: ', received_text);
+                // splt lines to accommodate for multiple commands
+                var lines = received_text.split("\n");
+                console.log('Lines read from NFC tag: ', lines);
+
+                for (const command of lines) {
+                  await process_sonos_command(command);
+                }
 
                 await process_sonos_command(received_text);
```

Comment:
I think this call is redundant (in the case of a single line) and an error (in the event of multiple lines.